### PR TITLE
Create a new contact UI

### PR DESCRIPTION
**Pull request by rishiip**
Fixes - #6 

SCR-20220824-sln
SCR-20220824-slt

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx [hunk 1]
```diff
@@ -292,3 +292,21 @@ export const ROWS = [
...
+    label: "Admin",
+    value: "admin",
+  },
+  {
+    label: "Agent",
+    value: "agent",
+  },
+];
+
+export const CONTACT_FORM_INITIAL_FORM_VALUES = {
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/constants.jsx:307]: So much redundancy in the variable name. Refactor.

Read: 
- https://academy.bigbinary.com/learn-react/pages/structuring-code#:~:text=Common%20mistakes%20done
- https://academy.bigbinary.com/learn-react/pages/naming-conventions-in-react

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+    } catch (err) {
+      logger.error(err);
+    }
+  };
+
+  return (
+    <Formik
+      initialValues={CONTACT_FORM_INITIAL_FORM_VALUES}
+      onSubmit={handleSubmit}
+    >
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:24]: Add the validations. Else adding a Formik Form is pointless.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+          <Button
+            className="mr-3"
+            icon={Check}
+            label="Save Changes"
+            size="large"
+            style="primary"
+            type="submit"
+            onClick={handleSubmit}
+          />
+          <Button label="Cancel" size="large" style="text" onClick={onClose} />
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:68]: https://academy.bigbinary.com/learn-react/pages/formik-forms#:~:text=Formik%20cancel%20button%20type

**Review by yedhink — COMMENTED** [review]
@rishiip Few comments added.

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx (hunk 1 above)
Comment (rishiip) [reply app/javascript/src/components/Dashboard/Contacts/constants.jsx:307]: This is done. Have a look - https://www.loom.com/share/34d73fa8665844119374c017739bb5e3

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 2 above)
Comment (rishiip) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:24]: This is done.

File: app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx (hunk 3 above)
Comment (rishiip) [reply app/javascript/src/components/Dashboard/Contacts/Pane/Form.jsx:68]: Thank you for suggestion. This is done.

**Comment by rishiip** [comment]
> @rishiip Few comments added.

@yedhink - Review modifications done. Have a look at PR #15

File: app/javascript/src/components/Dashboard/Contacts/constants.jsx (hunk 1 above)
Comment (yedhink) [reply app/javascript/src/components/Dashboard/Contacts/constants.jsx:307]: @rishiip Yes. You are correct. The new name looks good.

**Pull request merged**